src/main/java/Task_1/Application.java

```diff
@@ -0,0 +1,77 @@
+package hw8_1;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Application {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        Map<String, Person> people = new HashMap<>();
```

Comment:
Your code isn't compiled.
Please fix it in all places
![image](https://github.com/user-attachments/assets/8a6d8b98-47c0-451e-b122-6618d3ecfd9a)